### DISCUSSION TITLE
Why don't they add the meta=vierport?

### DISCUSSION THREAD

**Discussion by jlop77** [Lumieducation/Lumi#2060] [Lumi Run]
I'm exporting to html files, but I share them via WhatsApp or similar. Which you're going to see on mobile devices. I don't understand why it doesn't have the label:

```html
<meta name="viewport" content="user-scalable=no, width=device-width, initial-scale=1"> 
```

This is especially important when scaling texts. At the moment I export to html and then add the tag manually.

**Comment by sr258** [comment]
I've converted this into a feature request and I'll implement it soon. #2061 Thank you for the suggestion!

**Reply by jlop77** [reply]
Fenomenal, thx.
